frontend/src/modules/batchactions/actions.js

```diff
@@ -110,8 +110,12 @@ export function performAction(
             replace,
         );
 
-        const response: ResponseType = {};
-        response.action = action;
+        const response: ResponseType = {
+            changedCount: 0,
+            invalidCount: 0,
+            error: false,
+            action,
+        };
```

Comment:
Meh, my previous attempt to fix this didn't work, so I'm just starting off with a full object literal.

The alternative would be to do a bunch of `let`, and then create the response as `{action, changedCount, invalidCount, error}`. But I feel that's harder to read than this?